source/chapters/library.rst

```diff
@@ -837,7 +837,6 @@ view, then right-click on a session's name/date to access the different features
 * **Export playlist**: Export a session in various file formats. This allows you
   to use the data in other applications.
 
-
```

Comment:
Unrelated change. I suggest to use `git add -p` to avoid this in the future.

Reply:
This is the command I was looking for.  Thank you Jan.